recipes-core/system-api-fake-fifo/system-api-fake-fifo.bb

```diff
@@ -1,16 +1,8 @@
 DESCRIPTION = "Create a regular file to accumulate system events before System API starts"
 LICENSE = "CLOSED"
-FILESEXTRAPATHS:prepend := "${THISDIR}:"
-
-SRC_URI += "file://init"
-
-INITSCRIPT_NAME = "system-api-fake-fifo"
-INITSCRIPT_PARAMS = "defaults 60"
-
-inherit update-rc.d
 
 do_install() {
-    install -m 0755 -d ${D}/var/volatile
+    install -m 1777 -d ${D}/var/volatile
     touch ${D}/var/volatile/system-api.fifo
     chmod 666 ${D}/var/volatile/system-api.fifo
 }
```

Comment:
Don't forget to add the QA of creating such a file like this

FILES:${PN} += "<path-to-your-file>/<file-name: system-api.fifo>"